fieldservice_agreement/views/agreement_view.xml

```diff
@@ -45,4 +44,34 @@
         </field>
     </record>
 
+    <record id="agreement_search_fields" model="ir.ui.view">
+        <field name="name">agreement.search.fields</field>
+        <field name="model">agreement</field>
+        <field name="type">search</field>
+        <field name="inherit_id" ref="agreement.partner_agreement_search_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="/search/filter[1]" position="before">
+                <field name="name"/>
+                <field name="partner_id"/>
+                <field name="agreement_type_id"/>
+                <field name="agreement_subtype_id"/>
```

Comment:
Those 4 fields are not related to fieldservice. They should be in the original view in the agreement module.

Reply:
@max3903 : Change has been made here: https://github.com/OCA/contract/pull/279